### PR TITLE
Uniform batch scripts as presented in guide

### PR DESCRIPTION
**Pull request by KmaTTi89**
- edited build.bat, run.bat, test.bat, package.bat
- created package.bat

### CODE REVIEW

**Comment by KmaTTi89** [comment]
I don't have these files on local branch. It is weird

**Comment by KmaTTi89** [comment]
I don't see these files on my branch on github.com too

**Comment by warpech** [comment]
I am sorry, it is not wierd, I was being blind. I looked at `master` branch, not at `develop`.

**Comment by KmaTTi89** [comment]
Ok, can I merge?

File: test.bat [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
-star.exe --resourcedir="%~dp0test\Launcher_AcceptanceHelperOne\wwwroot" "%~dp0test\Launcher_AcceptanceHelperOne\bin\Debug\Launcher_AcceptanceHelperOne.exe"
-star.exe --resourcedir="%~dp0test\Launcher_AcceptanceHelperTwo\wwwroot" "%~dp0test\Launcher_AcceptanceHelperTwo\bin\Debug\Launcher_AcceptanceHelperTwo.exe"
+IF "%CONFIGURATION%"=="" SET CONFIGURATION=Debug
 
+SET INTERACTIVE=1
+ECHO %CMDCMDLINE% | find /i "%~0" >nul
+IF NOT ERRORLEVEL 1 SET INTERACTIVE=0
+
+:: Stop all the running apps
+staradmin stop db default
```
Comment (warpech) [inline test.bat]: `staradmin stop db default` is entered twice in this file

Comment (warpech) [inline test.bat]: I see that this is actually also a mistake in the Guidelines file. I will fix it there.

Comment (KmaTTi89) [inline test.bat]: Fixed

**Comment by warpech** [comment]
You are right, these files are already removed in the `develop` branch. I was looking at the `master`.

**Pull request merged**